.goreleaser.docker.yml

```diff
@@ -30,18 +30,22 @@ builds:
     ldflags: *ldflags
 
 dockers:
+  # AMD64
   - image_templates:
-      - &amd_image "quay.io/authzed/zed:{{ .Version }}-amd64"
-    ids: ["linux-amd64"]
+      - &amd_image_quay "quay.io/authzed/zed:v{{ .Version }}-amd64"
+      - &amd_image_gh "ghcr.io/authzed/zed:v{{ .Version }}-amd64"
+      - &amd_image_dh "authzed/zed:v{{ .Version }}-amd64"
     dockerfile: &dockerfile "Dockerfile.release"
     goos: "linux"
     goarch: "amd64"
     use: "buildx"
     build_flag_templates:
       - "--platform=linux/amd64"
+  # ARM64
   - image_templates:
-      - &arm_image "quay.io/authzed/zed:{{ .Version }}-arm64"
-    ids: ["linux-arm64"]
```

Comment:
I think you need to leave these id fields

Reply:
there's only 1 `build` in spicedb, but we have to have different ones here since it uses CGO